src/controllers/OrderController.js

```diff
@@ -0,0 +1,13 @@
+import { order } from '../models/store';
+
+class OrderController {
+  setOrder(request, response) {
+    request.body.datatime = new Date();
+    request.body.status = 'pending';
+    request.body.id = Math.floor(Math.random() * 10000000000000000000000000).toString();
```

Comment:
Do this have to be unique? Whereas it might seem impossible, this doesn't allow for unique ids. Please change this to use something more practical.